client/tx_config.go

```diff
@@ -30,6 +30,7 @@ type (
 		NewTxBuilder() TxBuilder
 		WrapTxBuilder(sdk.Tx) (TxBuilder, error)
 		SignModeHandler() *txsigning.HandlerMap
+		SigningContext() *txsigning.Context
```

Comment:
Should we just expose AddressCodec here instead of the entire signingContext leaking up?

Reply:
I don't think it hurts to leak it